Source/DafnyLanguageServer/Workspace/ProjectManager.cs

```diff
@@ -290,18 +290,18 @@ public class ProjectManager : IDisposable {
       compilation.IncrementJobs();
       var resolution = await compilation.Resolution;
 
-      var verifiables = resolution.CanVerifies?.ToList();
-      if (verifiables == null) {
+      var canVerifies = resolution.CanVerifies?.ToList();
```

Comment:
Seem like a weird name after all. Did you mean "VerifiableDeclarations"?

Reply:
That is what it means. CanVerifies is meant to the pural of CanVerify. Earlier I named these Verifiable and Verifiables. I'm OK with changing to `VerifiableDeclarations` although it is two words instead of one.